tools/semgrep-py.yml

```diff
@@ -43,6 +43,15 @@ rules:
       exclude:
         - "**/migrations/"
 
+  - id: dont-use-empty-select_related
+    pattern-either:
+      - pattern: $X.objects. ... .select_related()
+      - pattern: $X.objects. ... .prefetch_related()
```

Comment:
Any reason not to broaden this to simply `$X.select_related()` and `$X.prefetch_related()`?

Reply:
Nope.  👍🏻